components/branding-preference-mgt/org.wso2.carbon.identity.branding.preference.mgt/src/main/java/org/wso2/carbon/identity/branding/preference/mgt/util/BrandingPreferenceMgtUtils.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.branding.preference.mgt.util;
+
+import org.apache.commons.lang.StringUtils;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.wso2.carbon.identity.branding.preference.mgt.exception.BrandingPreferenceMgtClientException;
+import org.wso2.carbon.identity.branding.preference.mgt.exception.BrandingPreferenceMgtServerException;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Util class for branding preference management.
+ */
+public class BrandingPreferenceMgtUtils {
+
+    /**
+     * Generate Branding Preference input stream.
+     *
+     * @param preferencesJSON JSON string of preferences.
+     * @return Input stream of the preferences JSON string.
+     * @throws UnsupportedEncodingException Unsupported Encoding Exception.
+     */
+    public static InputStream generatePreferenceInputStream(String preferencesJSON) throws
+            UnsupportedEncodingException {
+
+        return new ByteArrayInputStream(preferencesJSON.getBytes(StandardCharsets.UTF_8.name()));
+    }
+
+    /**
+     * Check whether the given string is a valid JSON or not.
+     *
+     * @param stringJSON Input String.
+     * @return True if the input string is a valid JSON.
+     */
+    public static boolean isValidJSONString(String stringJSON) {
+
+        if (StringUtils.isBlank(stringJSON)) {
+            return false;
+        }
+        try {
+            JSONObject objectJSON = new JSONObject(stringJSON);
+            if (objectJSON.length() == 0) {
+                return false;
+            }
+        } catch (JSONException exception) {
+            return false;
```

Comment:
Would it be better to add a comment and a debug log here?

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/3870/commits/d8b98f955a8d30d3fb07768ec4b1e5503aa46a48